packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -20,13 +20,18 @@ class SessionLogManager {
 
   final LogSettings Function(Session) _settingsForSession;
 
+  int _logOrderId;
+
+  int get _nextLogOrderId => ++_logOrderId;
```

Comment:
Note(Non blocking): Commit message here is wrong right? It is moved into the log_manager and not the session?

Reply:
yes indeed